python/dgl/graphbolt/impl/ondisk_dataset.py

```diff
@@ -498,13 +499,18 @@ def _init_tvt_set(
 
     def _init_all_nodes_set(self, graph) -> Union[ItemSet, ItemSetDict]:
         if graph is None:
+            warnings.warn(
+                "Initializing `all_nodes_set` with a graph whose type does "
+                "not belong to SamplingGraph",
+                Warning,
+            )
             return None
         num_nodes = graph.num_nodes
         if isinstance(num_nodes, int):
-            return ItemSet(num_nodes)
+            return ItemSet(num_nodes, names="seed_nodes")
```

Comment:
any dedicated testcases for this change?

Reply:
Added just now.